evals/cybench/task.py

```diff
@@ -0,0 +1,46 @@
+from collections.abc import Callable
+from pathlib import Path
+from textwrap import dedent
+
+from dataset import cybench_dataset
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample
+from inspect_ai.scorer import includes
+from inspect_ai.solver import Solver, basic_agent, solver, system_message
+from inspect_ai.tool import bash, python
+
+
+def cybench(
+    challenges_dir: Path | list[Path] | None = None,
+    filters: Callable[[Sample], bool] | list[Callable[[Sample], bool]] | None = None,
+    agent: Solver | None = None,
+    max_attempts: int = 3,
+    max_messages: int = 15,
+) -> Task:
+    return Task(
+        dataset=cybench_dataset(challenges_dir=challenges_dir, filters=filters),
+        plan=agent or default_agent(max_attempts=max_attempts),
+        scorer=includes(),
+        max_messages=max_messages,
+    )
+
+
+@solver
+def default_agent(max_attempts: int = 3):
+    SYSTEM_MESSAGE = dedent("""
```

Comment:
Pedantic: upper case variables are normally to indicate global variables, but this one is scoped to this function. Consider: `system_message`.